src/partials/advanteges.html

```diff
@@ -1,6 +1,6 @@
 <section class="section">
     <div class="container">
-    <h2 class="advantages-h2-hidden">HIDDEN</h2>
+    <h2 class="visually-hidden">HIDDEN</h2>
```

Comment:
zmien nazwe na Advantages, bo to jest nagłówek sekcji

Reply:
już